plasmapy/physics/distribution.py

```diff
@@ -58,12 +57,11 @@ def Maxwellian_1D(v: units.m/units.s,
     In one dimension, the Maxwellian distribution function for a particle of
     mass m, velocity v, a drift velocity V and with temperature T is:
 
-    .. math::
-    f = \sqrt{\frac{m}{2 \pi k_B T}} \exp(-\frac{m}{2 k_B T} (v-V)^2)
+    :math:`f = \sqrt{\frac{m}{2 \pi k_B T}} e^{-\frac{m}{2 k_B T} (v-V)^2}`
```

Comment:
I'm noticing that `:math:` seems not to center the formulas. I'm guessing `.. math ::` would do that, thus the two would be somewhat equivalent to LaTeX's `$` and `$$` envelopes. Could you try this with `.. math ::` one more time, maybe google would help? I seem to recall seeing this `.. math ::` other libraries' documentation, so I think we might have been doing something else wrong.